solana/program_v2/package.json

```diff
@@ -35,7 +35,8 @@
     "mocha": "^10.0.0",
     "prettier": "^2.7.1",
     "ts-mocha": "^10.0.0",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@solana/spl-token": "^0.3.5"
```

Comment:
This should not be in the root package, right? :)

Reply:
the `scripts` (to generate the fixtures) were moved to the root... I added a "mint" generation for us to use for testing... hence needing the `spl-token` lib

Reply:
Shouldn't the fixture generation be in the tests package? As mentioned before I'd like to root to be as this as possible.